workflow/tools/util_in_docker/generate_song_payload.py

```diff
@@ -37,7 +42,16 @@ def main():
         analysis_type=get_analysis_type(yaml_data),
         info={
             'isPcawg': get_analysis_info_ispcawg(),
-            'dataset': get_analysis_info_dataset()
+            'dataset': get_analysis_info_dataset(),
+            'workflow': {
```

Comment:
should `workflow` be assigned by calling the function `get_workflow_data` which is defined below, but not called anywhere.